### PR TITLE
chore: Project focus updated by Harshu467 #1884

### PR DESCRIPTION
**Pull request by Harshu467**
## Project Focus 1884

As mentioned in issue number #1884 about project Focus effect, Updated

## Description

I have updated design in profile section sharing sample

## Screenshots

[Screencast from 30-07-23 11:25:57 AM IST.webm](https://github.com/priyankarpal/ProjectsHut/assets/89975347/6e2c6b1c-1448-4722-bb3d-c4968890dd53)

### CODE REVIEW

**Comment by Harshu467** [comment]
@priyankarpal Please check and merge and give labels of gsoc and level , Thank you

File: app/projects/[username]/page.tsx [hunk 1]
```diff
@@ -223,7 +223,7 @@ function ProjectList() {
             Object.keys(userObj).length > 0 &&
             userObj?.Projects.map((project, index) => (
               <div
-                className="w-100 my-1 p-4 mb-4 text-white border border-gray-800 "
+                className="w-100 my-1 p-4 mb-4 text-white cursor-pointer hover:border-primary border border-gray-800 "
```
Comment (priyankarpal) [inline app/projects/[username]/page.tsx:226]: remove `cursor-pointer` 
```suggestion
                className="w-100 my-1 p-4 mb-4 text-white hover:border-primary border border-gray-800 "
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: app/projects/[username]/page.tsx (hunk 1 above)
Comment (Harshu467) [reply app/projects/[username]/page.tsx]: Updated Please check and merged

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**